README.md

```diff
@@ -0,0 +1,217 @@
+#PolyPasswordHasher-ECC
+
+An extension of the [PolyPasswordHasher password storage scheme](https://github.com/PolyPasswordHasher/PolyPasswordHasher). 
+This repository uses Reed Solomon error correcting codes instead of vanilla Shamir Secret Sharing to manage a polypasswordhasher store.
+
+More details about the original scheme can be found [here](https://github.com/PolyPasswordHasher/PolyPasswordHasher).
+This version enables an error-correcting threshold-cryptosystem by using both errors and erasures decoding. We use the Python library [unireedsolom 1.0](https://pypi.python.org/pypi/unireedsolomon)
+for implementing the same. This allows improved performance of the server when validating accounts as some number of incorrect passwords
```

Comment:
I think it would be clearer to say:
```
This version enables an error-correcting threshold-cryptosystem by using both errors and erasures decoding with unireedsolomon 1.0. This provides better performance for the server when validating accounts as some number of incorrect passwords are allowed for successful secret recovery. Adding to this, the life of the attacker is also harder, because this scheme exherts more effort at higher error rates
```

Or something of the like :+1: 

In general, I like the structure.